core/src/main/java/org/apache/accumulo/core/client/admin/TableOperations.java

```diff
@@ -433,6 +433,33 @@ void clone(String srcTableName, String newTableName, boolean flush,
       Map<String,String> propertiesToSet, Set<String> propertiesToExclude) throws AccumuloException,
       AccumuloSecurityException, TableNotFoundException, TableExistsException;
 
+  /**
+   * Clone a table from an existing table. The cloned table will have the same data as the source
+   * table it was created from. After cloning, the two tables can mutate independently. Initially
+   * the cloned table should not use any extra space, however as the source table and cloned table
+   * major compact extra space will be used by the clone.
+   *
+   * Initially the cloned table is only readable and writable by the user who created it.
+   *
+   * @param srcTableName
+   *          the table to clone
+   * @param newTableName
+   *          the name of the clone
+   * @param flush
+   *          determines if memory is flushed in the source table before cloning.
+   * @param propertiesToSet
+   *          the sources tables properties are copied, this allows overriding of those properties
+   * @param propertiesToExclude
+   *          do not copy these properties from the source table, just revert to system defaults
+   * @param keepOffline
+   *          do not bring the table online after cloning
+   */
```

Comment:
need a since 2.1 javadoc tag

Reply:
I think he is targeting the 1.9 branch, which will be 1.10.  So I guess it should be a since 1.10 javadoc tag, which may be the only one haha.

Reply:
Interesting, API in 1.10, not in 2.0, and in 2.1.

Reply:
so what should I do in this case?

Reply:
I am not sure... maybe it could be `@since 1.10 2.1.0`

Reply:
`@since 1.10.0 and 2.1.0` works. Or, since we don't generally add new APIs in bugfix/patch releases, it might suffice to put `@since 1.10 and 2.1`

Reply:
will do